.github/workflows/Repology.yml

```diff
@@ -12,8 +12,8 @@ jobs:
         run: |
             sudo docker pull satmandu/crewbuild:amd64
             sudo docker run -t -v $(pwd):/usr/local/json satmandu/crewbuild:amd64 sudo -i -u chronos /bin/bash -c "
-            crew update
-            ruby ../tools/json.rb
+            crew update &&
+            ruby ../tools/json.rb &&
             cp repology.json /usr/local/json"
```

Comment:
Does this need to be
```
crew update && \
ruby ../tools/json.rb && \
cp repology.json /usr/local/json"
```

Reply:
It'll work without them

Reply:
Just so you know, the `&&` means if any command fails the following commands will not execute.

Reply:
In this case I'm just using it to separate the commands so crew doesn't misunderstand. Although, the behaviour of not continuing from a fail is probably desired here.